docs/user-guide/configuration-directories.md

```diff
@@ -0,0 +1,12 @@
+# Configuration Directories
+When running, the App Server will access the server's settings and read or modify the contents of its resource storage. All of this data is stored within a hierarchy of folders which correspond to scopes:
```

Comment:
try to avoid using "will." Please rephrase to something like "... the App Server accesses the server's ... "